glslc/test/working_directory.py

```diff
@@ -62,6 +62,31 @@ class TestWorkDirEqNoArgLinkFile(expect.ValidNamedObjectFile):
     # subdir/, where the source is.
     expected_object_filenames = ('a.spv',)
 
+
+@inside_glslc_testsuite('WorkDir')
+class TestMultipleWorkDir(expect.ValidNamedObjectFile):
+    """Tests that if there are multiple -working-directory=<dir> specified,
+    only the last one takes effect."""
+
+    environment = Directory('.', [EMPTY_SHADER_IN_SUBDIR])
+    glslc_args = ['-c', '-working-directory=i-dont-exist',
+                  '-working-directory', 'i-think/me-neither',
+                  '-working-directory=subdir', 'shader.vert']
+    # Output file should be generated into subdir/.
+    expected_object_filenames = ('subdir/shader.vert.spv',)
+
+
+@inside_glslc_testsuite('WorkDir')
+class TestWorkDirPosition(expect.ValidNamedObjectFile):
+    """Tests that -working-directory=<dir> affects all files including the
```

Comment:
But it doesn't test "all files" -- just one.  We don't currently have an all-files test; I'll add one.